### PR TITLE
Prepare release of ghcide 0.3.0

### LINKED ISSUE #719: Upload a new version to Hackage
It's been two months since 0.2.0 and there's been a ton of improvements and version upgrades. The upgrade to Hie-bios 0.6.1 alone brings nice improvements for Bios cradles which make things much easier in large Buck multi-repos like ours.

### PR DESCRIPTION
**Pull request by pepeiborra**
Closes #719

### CODE REVIEW

File: ghcide.cabal [hunk 1]
```diff
@@ -2,17 +2,17 @@ cabal-version:      1.20
...
 name:               ghcide
-version:            0.2.0
+version:            0.3.0
 license:            Apache-2.0
 license-file:       LICENSE
-author:             Digital Asset
-maintainer:         Digital Asset
+author:             Digital Asset + Ghcide contributors
+maintainer:         Ghcide contributors
 copyright:          Digital Asset 2018-2020
```
Comment (ndmitchell) [inline ghcide.cabal:10]:
```suggestion
copyright:          Digital Asset and Ghcide contributors 2018-2020
```

That worth changing?

**Review by ndmitchell — APPROVED** [review]
Looks good!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,62 @@
...
+* Backport HIE files to GHC 8.6 (#689) - (wz1000)
+* Performance improvements for GetSpanInfo (#681) - (Pepe Iborra)
+* Code action add default type annotation to remove `-Wtype-defaults` warning (#680) - (Serhii)
+* Use a global namecache to read `.hie` files (#677) - (wz1000)
+* Completions need not depend on typecheck of the current file (#670) - (Pepe Iborra)
+* Fix spaninfo Haddocks for local modules (#678) - (Pepe Iborra)
+* Avoid excessive retypechecking of TH codebases (#673) - (Pepe Iborra)
+* Use stale information if it's available to answer requests quickly (#624) - (Matthew Pickering)
+* Code action: add constraint (#653) - (Denis Frezzato)
+* Code action: add constraint (#653) - (Denis Frezzato)
```
Comment (ocharles) [inline CHANGELOG.md:29]:
```suggestion
* Code action: add constraint (#653) - (Denis Frezzato)
```

Comment (pepeiborra) [reply CHANGELOG.md:29]: Oops, the changelog is auto generated from `git log` so I'm not sure how this duplicated commit ended up there!

**Pull request merged**